chapters/data-collection.tex

```diff
@@ -1,362 +1,286 @@
+anyone without access to a specific deciphering 
+key. \url{https://dimewiki.worldbank.org/wiki/Encryption}}
+all data submitted from the field while in transit (i.e., upload or download).  Your data will be encrypted from the time it leaves the device (in tablet-assisted data collation) or your browser (in web data collection), until it reaches the server. Therefore, as long as you are using an established CAPI software, this step is largely taken care of. However, the research team must ensure that all computers, tablets, and accounts used have a logon password and are never left unlocked.
+
+\subsection{Secure data storage}
+\textbf{Encryption at rest} is the only way to ensure that PII data remains private when it is stored on someone else's server on the open internet. You must keep your data encrypted on the server whenever PII data is collected.
+Encryption makes data files completely unusable without access to a security key specific to that data -- a higher level of security than password-protection.
+Encryption at rest requires active participation from the user; and you should be fully aware that if your private key is lost, there is absolutely no way to recover your data.
+
+You should not assume that your data is encrypted by default: indeed, for most CAPI software platforms, encryption needs to be enabled by the user.
+To enable it, you must confirm you know how to operate the encryption system and understand the consequences if basic protocols are not followed.
+When you enable encryption, the service will allow you to download -- once -- the keyfile pair needed to decrypt the data.
+You must download and store this in a secure location, such as a password manager. Make sure you store keyfiles with descriptive names to match the survey to which they correspond.
+Any time anyone accesses the data - either when viewing it in the browser or downloading it to your computer - they will be asked to provide the keyfile.
+Only project teams members names in the IRB are allowed access to the private keyfile.
+
+To proceed with data analysis, you typically need a working copy of the data accessible from a personal computer. The following workflow allows you to receive data from the server and store it securely, without compromising data security.
+
+\begin{enumerate}
+	\item Download data
+	\item Store a ``master'' copy of the data into an encrypted location that will remain accessible on disk and be regularly backed up
+	\item Secure a ``gold master'' copy of the raw data in a secure location, such as a long-term cloud storage service or an encrypted physical hard drive stored in a separate location. If you remain lucky, you will never have to access this copy -- you just want to know it is out there, safe, if you need it.
+
+\end{enumerate}
+
+This handling satisfies the rule of three: there are two on-site copies of the data and one off-site copy, so the data can never be lost in case of hardware failure.
```

Comment:
Is the _rule of three_ the same as what we call the _3-2-1 rule_ in chapter 2:

https://github.com/worldbank/d4di/blob/17f55ca77a6e5b8ae594ea00c6fe1ad3926a861b/chapters/planning-data-work.tex#L91

Reply:
@bbdaniels ?

Reply:
linking to #183 as it is also discussed there

Reply:
either wording is fine with me

Reply:
issue #332